pkg/operator/pdf/v0/convertPdfToMarkdown_test.go

```diff
@@ -0,0 +1,51 @@
+package pdf
+
+import (
+	"encoding/base64"
+	"os"
+	"testing"
+)
+
+func TestConvertPdfToText(t *testing.T) {
+	tests := []struct {
+		name     string
+		filepath string
+	}{
+		{
+			name:     "Convert pdf file",
+			filepath: "testdata/test.pdf",
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			b, err := os.ReadFile(test.filepath)
```

Comment:
If you're not going to check the output of the conversion, you can just pass some random bytes here